clmm/tests/test_galaxycluster.py

```diff
@@ -0,0 +1,75 @@
+import os, sys
+DIR = os.path.dirname(os.path.realpath(__file__))
+sys.path.append('/'.join(DIR.split('/')[:-1]))
+
+from galaxycluster import *
+from datatypes import *
+
+test_creator = 'Mitch'
+test_creator_diff = 'Witch'
+
+test_dict = {'test%d'%i:True for i in range(3)}
+test_dict_diff = {'test%d'%i:False for i in range(3)}
+test_dict_sub = {'test%d'%i:True for i in range(2)}
+test_data = GCData(test_creator, test_dict,[1, 2, 3])
+
+from numpy import testing as tst
+
+def test_check_subdict():
+
+    tst.assert_equal(True, check_subdict(test_dict_sub, test_dict))
```

Comment:
`assert check_subdict(test_dict_sub, test_dict)`
`assert not check_subdict(test_dict, test_dict_sub)`